README.md

```diff
@@ -22,6 +22,12 @@ require 'chef_handler_foreman'
 foreman_server_options  :url => 'http://your.server/foreman'
 # add following line if you want to upload node attributes (facts in Foreman language)
 foreman_facts_upload    true
+# add following line if you want to upload only specific node attributes
```

Comment:
could you please mention that only top-level attributes can be specified here (or at least that's my understanding from the code), ideally also below in `## Facts whitelist / blacklisting`, blacklist seems to support any attribute and is specified as a full name